stories/TextInputPagination.js

```diff
@@ -10,6 +11,7 @@ const TextInputPagination = ({ currentPage, inputtedPage, totalPages, nextPage,
 
     return(
         <div className="react-collapsible-page">
+            <span>Showing { (currentPage - 1) * rowSize + 1}-{rowSize * currentPage } of {totalResults}</span>
```

Comment:
what if not all users wants to show the totalResults? 
Need to consider this library is/will be using many projects. 

I think this should be embedded in loop. need to think how can be done.